apps/openmw/mwgui/tradewindow.hpp

```diff
@@ -65,8 +65,10 @@ namespace MWGui
 
             int mItemToSell;
 
-            int mCurrentBalance;
-            int mCurrentMerchantOffer;
+            bool mIsPlayerSelling;
+            bool mIsMerchantBuying;
```

Comment:
Can you elaborate why we need these flags? I think the old way of using the sign to indicate buying/selling works fine. And if the price is 0, then buying or selling makes no difference anyway.